f8a_worker/storages/postgres.py

```diff
@@ -231,7 +231,8 @@ def store_api_requests_post(self, arguments):
             ecosystem=arguments['data'].get('ecosystem'),
             user_agent=arguments['data'].get('user_agent'),
             stack_data=json.dumps(arguments['data'].get('packages_list')),
-            manifest_hash=arguments['data'].get('manifest_hash')
+            manifest_hash=arguments['data'].get('manifest_hash'),
+            source=arguments['data'].get('source')
```

Comment:
Are you planning to add the redhat_commons_id later?

Reply:
I would prefer telemetry_id.

Reply:
Yes.first I will modify the api-server then add it here.